broker-daemon/state-machines/plugins/abstract.js

```diff
@@ -9,8 +9,8 @@ class StateMachineAbstractPlugin {
   constructor () {} // eslint-disable-line
 
   /**
-   * A copy of StateMachine~plugin, which is not exposed
-   * @param  {StateMachine} instance       - State machine instance to be hooked
```

Comment:
this seems like a step backward

Reply:
I added a typedef for this, not sure if it's right, but it passes